content/miq_dialogs/miq_provision_ibm_powervs_vm_dialog.yaml

```diff
@@ -33,6 +33,11 @@
           :display: :edit
           :default: 1
           :data_type: :integer
+        :placement_group:
+          :description: Placement Group
```

Comment:
Use the `:required_method:` key to reference your `validate_placement_group` method from the provision workflow.

Reply:
```suggestion
          :description: Placement Group
          :required_method: :validate_placement_group
```